src/Action.js

```diff
@@ -0,0 +1,28 @@
+export default class Action {
+	constructor (promise) {
+		this.promise = promise
+	}
+
+	// default onFulfilled action
+	/* istanbul ignore next */
+	fulfilled (p) {
+		this.promise._become(p)
+	}
+
+	// default onRejected action
+	rejected (p) {
+		this.promise._become(p)
+		return false
+	}
+
+	tryCall (f, x) {
```

Comment:
I found that I had to read this method a few times to convince myself it was doing what I expected it to do.  This seems more obvious to me:

```js
try {
  this.handle(f(x))
} catch(e) {
  this.promise._reject(e)
}
```

What do you think?